lib/RTCSession/ReferSubscriber.js

```diff
@@ -54,10 +54,13 @@ module.exports = class ReferSubscriber extends EventEmitter
 
     extraHeaders.push(referTo);
 
-    // Referred-By header field.
-    const referredBy = `Referred-By: <${this._session._ua._configuration.uri._scheme}:${this._session._ua._configuration.uri._user}@${this._session._ua._configuration.uri._host}>`;
+    // Referred-By header field (if not already present).
+    if (!extraHeaders.some(header => header.startsWith('Referred-By:'))) 
```

Comment:
This is wrong. It should not be case sensitive. SIP headers are not case sensitive.

Reply:
Then it should be `header.toLowerCase().startsWith('referred-by:')`. New PR or can you guys fix this in master?

Reply:
> SIP headers are not case sensitive

Absolutely right.

Fixed in new release.